client/Lykke.Service.PayInvoice.Client/PayInvoiceClient.cs

```diff
@@ -0,0 +1,205 @@
+        }
+
+        public async Task<InvoiceModel> GetInvoiceAsync(string invoiceId)
+        {
+            return await _runner.RunAsync(() => _invoicesApi.GetAsync(invoiceId));
+        }
+
+        public async Task<InvoiceDetailsModel> CheckoutInvoiceAsync(string invoiceId)
+        {
+            return await _runner.RunAsync(() => _invoicesApi.CheckoutAsync(invoiceId));
+        }
+
+        public async Task<IEnumerable<InvoiceModel>> GetInvoicesAsync(string merchantId)
+        {
+            return await _runner.RunAsync(() => _invoicesApi.GetAllAsync(merchantId));
+        }
+
+        public async Task<InvoiceModel> GetInvoiceAsync(string merchantId, string invoiceId)
+        {
+            return await _runner.RunAsync(() => _invoicesApi.GetByIdAsync(merchantId, invoiceId));
+        }
+
+        public async Task<InvoiceModel> CreateDraftInvoiceAsync(string merchantId, CreateDraftInvoiceModel model)
+        {
+            var content = new StringContent(JsonConvert.SerializeObject(model), Encoding.UTF8, "application/json");
+            var result = await _httpClient.PostAsync($"/api/merchants/{merchantId}/invoices/drafts", content);
+
+            if (result.IsSuccessStatusCode)
+            {
+                string value = await result.Content.ReadAsStringAsync();
+                return JsonConvert.DeserializeObject<InvoiceModel>(value);
+            }
+
+            throw new ErrorResponseException("An error occurred  during calling api");
+
+            //return await _runner.RunAsync(() => _invoicesApi.CreateDraftAsync(merchantId, model));
+        }
+
+        public async Task UpdateDraftInvoiceAsync(string merchantId, string invoiceId, CreateDraftInvoiceModel model)
+        {
+            var content = new StringContent(JsonConvert.SerializeObject(model), Encoding.UTF8, "application/json");
+            var result = await _httpClient.PutAsync($"/api/merchants/{merchantId}/invoices/{invoiceId}", content);
+
+            if (!result.IsSuccessStatusCode)
+                throw new ErrorResponseException("An error occurred  during calling api");
+            
+            //await _runner.RunAsync(() => _invoicesApi.UpdateDraftAsync(merchantId, invoiceId, model));
+        }
+
+        public async Task<InvoiceModel> CreateInvoiceAsync(string merchantId, CreateInvoiceModel model)
+        {
+            var content = new StringContent(JsonConvert.SerializeObject(model), Encoding.UTF8, "application/json");
+            var result = await _httpClient.PostAsync($"/api/merchants/{merchantId}/invoices", content);
+
+            if (result.IsSuccessStatusCode)
+            {
+                string value = await result.Content.ReadAsStringAsync();
+                return JsonConvert.DeserializeObject<InvoiceModel>(value);
+            }
+
+            throw new ErrorResponseException("An error occurred  during calling api");
+            //return await _runner.RunAsync(() => _invoicesApi.CreateAsync(merchantId, model));
```

Comment:
Please remove comments

Reply:
It's Okay, I hope Refit will be updated and I can use it this case